libs/shared/ui/src/lib/components/inputs/input-select/utils/change-select-value.ts

```diff
@@ -0,0 +1,21 @@
+import { act, fireEvent, getByRole, getByTestId } from '@testing-library/react'
+
+export function simulateSpaceKeyClick(element: Element) {
```

Comment:
I think we have a problem with this file because we have two files of tests this and `change-select-value.ts`, it's not normal?